src/main/java/com/faforever/client/chat/UserInfoWindowController.java

```diff
@@ -368,56 +382,92 @@ private void enterAchievementsLoadedState() {
     achievementsPane.setVisible(true);
   }
 
-  public void ladder1v1ButtonClicked() {
-    loadStatistics(KnownFeaturedMod.LADDER_1V1);
+  public void onRatingTypeChange() {
+    CompletableFuture<Void> statisticsFuture = switch (ratingTypeComboBox.getValue()) {
+      case LADDER_1V1 -> loadStatistics(KnownFeaturedMod.LADDER_1V1);
+      default -> loadStatistics(KnownFeaturedMod.FAF);
```

Comment:
If you put this as enum instance variables as well u would only need to add a new enum if u had a third rating type :D